src/seedu/addressbook/data/person/address/Unit.java

```diff
@@ -1,6 +1,6 @@
 package seedu.addressbook.data.person.address;
 
-public class Unit extends AddressElement<String> {
+public class Unit extends AddressElement {
```

Comment:
Hi Jeremy, Your implementation of inheritance is not exactly wrong although as mentioned during tutorial its not exactly necessary at this point of time.   An abstract interface instead of class would actually be a more accurate version what you might be looking to implement instead. However, its not taught at this point of time, do keep a look out about it in upcoming lectures! :)